### PR TITLE
Add releases script and page

### LINKED ISSUE #10: expose documentation for previous releases
copied from https://github.com/ramda/ramda/issues/821

### PR DESCRIPTION
**Pull request by MattMS**
Fixes #10

### CODE REVIEW

File: make_releases_html.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+var R = require('ramda')
+
+var semver_compare = require('semver-compare')
+
+var version = require('./package.json').devDependencies.ramda
+
+var template = fs.readFileSync('releases.html.handlebars', {encoding: 'utf8'})
+
+var version_folder_re = /\d+\.\d+(\.\d+)?/
```
Comment (buzzdecafe) [inline make_releases_html.js]: could be even stronger with bounds, e.g. `/^\d+\.\d+(\.\d+)?$/`

Comment (MattMS) [inline make_releases_html.js]: Very true. Have fixed.

**Comment by MattMS** [comment]
This doesn't yet include a way to get to the releases page.
I'm not sure how people would prefer navigating to it.

Also undecided on how to navigate from previous versions to other versions.

**Comment by buzzdecafe** [comment]
> This doesn't yet include a way to get to the releases page.

that doesn't need to be part of this pr. i could see that being a link on the index

**Pull request merged**